sklearn/linear_model/sgd_fast.pyx

```diff
@@ -22,7 +22,7 @@ from numpy.math cimport INFINITY
 cdef extern from "sgd_fast_helpers.h":
     bint skl_isfinite(double) nogil
 
-from sklearn.utils.weight_vector cimport WeightVector
+from sklearn.utils.weight_vector cimport WeightVector64 as WeightVector
```

Comment:
This should not be here but in the template. Maybe we have some sort of clash. I need to get back to that

Reply:
unless you also template `sgd_fast`, you'll have to deal with both, and switch with
```python
if floating is float:
    do something with WeightVector64
else:
    do something with WeightVector32
```

Reply:
Also this diff dong belong to this PR